app/assets/stylesheets/fonts.scss

```diff
@@ -1,19 +1,43 @@
 $brand: "Comfortaa", "Helvetica", sans-serif;
+$body-text: "Fira Sans", "Helvetica", sans-serif;
 
 .monospace{
   font-family: "Fira Mono", monospace;
   font-size: 1.1rem;
 }
 
+.brand {
+  font-family: $brand;
+}
+
+
+
 html {
-  font-family: "Fira Sans", "Helvetica", sans-serif;
+  font-family: $body-text;
 }
 
-.brand {
-  font-family: $brand;
+h1 {
+  @extend .brand;
+  text-align: center;
+}
+h2 {
+  @extend .dark-text;
+}
+h3 {
+  @extend .dark-text;
+}
+h4{
+  @extend .dark-text;
+}
+h5{
+
+}
+h6{
+  
 }
 
 p {
+  @extend .dark-text.secondary;
```

Comment:
I think it made sense to have only text in `p` tags and opacity of 0.87 is what is suggested by the material design documentation. Using the secondary of 0.54 opacity might be a bit too low in my opinion so if you think that is too low as well I can switch it up to 0.87. Alternatively, I'll make them a specific grey value instead of tweaking opacity.